packages/core-modules/contracts/storage/NftStorage.sol

```diff
@@ -4,7 +4,6 @@ pragma solidity ^0.8.0;
 contract NftStorage {
     struct NftStore {
         bool initialized;
-        address mainProxy;
```

Comment:
Not needed anymore. The account system knows who the system is because it should be its owner.